tofhir-server/src/main/scala/io/tofhir/server/endpoint/CodeSystemEndpoint.scala

```diff
@@ -48,22 +48,22 @@ class CodeSystemEndpoint extends LazyLogging {
     }
   }
 
-  /**
-   * Route to create a code system within a terminology
-   *
-   * @return
-   */
-  private def createCodeSystemRoute(terminologyId: String): Route = {
-    post {
-      entity(as[TerminologyCodeSystem]) { codeSystem =>
-        complete {
-          service.createCodeSystem(terminologyId, codeSystem) map { created =>
-            StatusCodes.Created -> created
-          }
-        }
-      }
-    }
-  }
+//  /**
+//   * Route to create a code system within a terminology
+//   * TODO: Implement job terminology update feature before using this
```

Comment:
What does that mean ? I think that feature is already implemented in this MR ?

Reply:
I don't think this much commented code is useful. We can simply remove them instead of merging them like this. We can retrieve it from git history if needed in the future.